workspaces/ui/src/services/TestingService.js

```diff
@@ -144,6 +166,24 @@ export async function createExampleTestingService(exampleId = 'todo-report') {
   return new ExampleTestingService(orgId);
 }
 
+export class TestingServiceError extends Error {
+  static type = 'testing-service-error';
```

Comment:
Would it be better to use a Symbol here?

Reply:
You could if type should be private, but exposing `type` as string on Error seems to be becoming the defect way in the wider ecosystem, so 🤷‍♂️.

Reply:
I think i'd do both, use symbol for checking so you don't have to worry about collisions, type string for printing, and since I presumably have to fix it in our typescript code in several places makes sense to make it a generic pattern.

Reply:
I'll soon be porting this to Typescript so the mock and production implementation can share an interface, so let's tackle this then.